### PR TITLE
added myself

### PR DESCRIPTION
**Pull request by 8D1**

### CODE REVIEW

File: contributors.json [hunk 1]
```diff
@@ -663,4 +663,14 @@
 			"YARG": [ "Developer" ]
 		}
 	}
+		{
```
Comment (EliteAsian123) [inline contributors.json]: You have an extra tab here, and you're missing the comma at the end of the previous line.

**Review by EliteAsian123 — CHANGES_REQUESTED** [review]
Thanks for the PR! Once the formatting is fixed this should be good to go.

**Comment by 8D1** [comment]
Should be fixed, Sorry about that!

**Review by EliteAsian123 — APPROVED** [review]
No problem!

**Pull request merged**